components/rdf-load.js

```diff
@@ -3,53 +3,48 @@
 var _ = require('underscore');
 var Promise = require('promise');
 var rdfstore = require('rdfstore');
-var noflo = require('noflo');
 
 var basenode = require('./base-node');
+var promiseComponent = require('./promise-component');
 
-exports.getComponent = function() {
-    return _.extend(new noflo.Component({
-        outPorts: {
-            out: {
-                description: "All the quads as a RDF JS Interface Graph object",
-                datatype: 'object'
-            },
-            error: {
-                description: "Error object",
-                datatype: 'object'
-            }
+exports.getComponent = promiseComponent({
+    description: "Loads data into a RDF JS Interface Graph object",
+    icon: 'sign-in',
```

Comment:
The imagery of an arrow pointing into a box seemed appropriate for this parsing functionality. I thought the upload/download icons carried too much semantic weight.